packages/image/src/image.tsx

```diff
@@ -25,14 +35,19 @@ export const Image = forwardRef<ElementRef<typeof defaultTag>, ImageProps>(
     }) ?? { src: imagePlaceholderSvg };
 
     return (
-      <img {...imageProps} {...imageAttributes} decoding="async" ref={ref} />
+      <img
+        {...imageProps}
+        {...imageAttributes}
+        decoding={decoding}
+        loading={loading}
```

Comment:
shoulnd't these be overwritable by props?

Reply:
its props just with defaults ;) so will be overrided